python/tk_desktop2/action_handler.py

```diff
@@ -5,8 +5,10 @@
 # this software in either electronic or hard copy form.
 #
 
+import json
 import sgtk
 import time
+import pickle
```

Comment:
cPickle outperforms pickle, but can be problematic if you're communicating serialized data across platforms. In this case, since all communication is happening on localhost, it should be safe to use cPickle.